paraflow-metaserver/src/main/java/cn/edu/ruc/iir/paraflow/metaserver/utils/MetaConfig.java

```diff
@@ -33,7 +37,41 @@ public void init(String configPath) throws ConfigFileNotFoundException
     public boolean validate()
     {
         // TODO alice: validate configuration content
-        return true;
+        try {
+            //dbDriver format check
+            String dbDriver = getDBDriver();
+            String regExDriver = "^[a-zA-Z]+.[a-zA-Z]+.[a-zA-Z]+$";
+            Pattern pDriver = Pattern.compile(regExDriver);
+            Matcher mDriver = pDriver.matcher(dbDriver);
+            boolean driver = mDriver.find();
+            //dbHost format check
+            String dbHost = getDBHost();
+            String regExHost = "^[a-zA-Z]+:[a-zA-Z]+://[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}:[0-9]{1,5}(/[a-zA-Z]+)+$";
+            Pattern pHost = Pattern.compile(regExHost);
+            Matcher mHost = pHost.matcher(dbHost);
+            boolean host = mHost.find();
+            //dbUser format check
+            String dbUser = getDBUser();
+            String regExUser = "^[a-zA-Z][a-zA-Z0-9]*$";
+            Pattern pUser = Pattern.compile(regExUser);
+            Matcher mUser = pUser.matcher(dbUser);
+            boolean user = mUser.find();
+            //HDFSWarehouse format check
+            String hdfsWarehouse = getHDFSWarehouse();
+            String regExWarehouse = "^[a-zA-Z]+://[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}:[0-9]{1,5}(/[a-zA-Z]+)+$";
```

Comment:
host name may not contain ip address, for example:
hdfs://localhost:9000/example